bin/ci/powershell-linting.ps1

```diff
@@ -0,0 +1,21 @@
+# Ensure plan name has been passed
+param (
+    [Parameter()]
+    [string]$Plan = $(throw "Usage: powershell-linting.ps1 [plan] e.g. powershell-linting.ps1 7zip")
+)
+
+$ErrorActionPreference = "Stop"
+
+# Ensure PSScriptAnalyzer is installed
+if (-Not (Get-Module -ListAvailable -Name PSScriptAnalyzer)) {
+  # TODO: PSScriptAnalyzer should have its own core plan
```

Comment:
I'd imagine this would be super trivial and similar to the `pester` plan. I'd recommend either including it with this PR or adding an issue to track it and delete the TODO.

Reply:
Raised a PR for this https://github.com/habitat-sh/core-plans/pull/2807

It looks like it involves more work than Pester, unless I misunderstand Windows plans. I tagged you in a comment on that PR.